README.md

```diff
@@ -71,7 +81,7 @@ MODULE_TOPDIR to where to the GRASS binaries are located:
 make MODULE_TOPDIR=/usr/lib/grass78/
 ```
 
-## How to submit contributions
+### Contribute
```

Comment:
This could be changed to "Submiting guidelines" or so...

Reply:
Do you suggest then to remove this section from README.md completely, and move whatever is down here to CONTRIBUTING.md? If yes, it sounds reasonable to me. Moreover, the submitting link is also in SUBMITTING.md. So, we could just link to both files and done. I can create a new PR from scratch with all suggestions here.

Reply:
Sounds good.

Perhaps, you can remove SUBMITTING. The word for it now is _contributing_ and we have CONTRIBUTING.md. Will anyone look for a file called SUBMITTING and ignore file called CONTRIBUTING.md? Or is there an automated system which uses that? Probably not.